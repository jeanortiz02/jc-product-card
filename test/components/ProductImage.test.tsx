
import React from "react";
import { render } from "@testing-library/react";
import { ProductImage } from "../../src/components/ProductImage";
import { ProductCard } from "../../src";
import { product2 } from "../data/product";



describe('Pruebas en el ProductImage', () => {
    
    test('debe mostrar la imagen personalizada', () => {
        const { container } = render(
                    <ProductImage image="image-url"/>
                )
        
               expect(container).toMatchSnapshot();
    });
    test('debe mostrar la imagen del producto', () => {
        const { container } = render(
                   <ProductCard
                        product={product2}
        
                    >
                        {
                            ()=> (
                                <ProductImage />
                            )
                        }
                    </ProductCard>
                )
        
               expect(container).toMatchSnapshot();
    });
});