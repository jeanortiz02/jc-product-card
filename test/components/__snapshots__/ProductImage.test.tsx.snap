// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el ProductImage debe mostrar la imagen del producto 1`] = `
<div>
  <div
    class="productCard undefined"
  >
    <img
      alt="Product Image"
      class="productImg undefined"
      src="./coffee-mug.png"
    />
  </div>
</div>
`;

exports[`Pruebas en el ProductImage debe mostrar la imagen personalizada 1`] = `
<div>
  <img
    alt="Product Image"
    class="productImg undefined"
    src="image-url"
  />
</div>
`;
